### ISSUE TITLE
docs need an update

### ISSUE THREAD

**Issue by PaulSoderlind** [JuliaStats/HypothesisTests.jl#258]
It seems the docs have not been updated for a while. I think I helped out with Clark-West and White's test in 2020, but I cannot find them in the documentation. Or, did I miss something?

**Comment by nalimilan** [comment]
Good catch. Docs have not been deployed for ages, probably because of a missing DOCUMENTER_KEY. I'ved added keys, and when https://github.com/JuliaStats/HypothesisTests.jl/pull/259 is merged I'll check whether deployment worked.

**Comment by nalimilan** [comment]
It works now. Docs for development version include the new tests. Docs for stable version will be fixed when we tag the next release.
